GoMeet/src/main/webapp/create-meeting-step2-gcal.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <!-- Create Meeting Step 2 GCal: Automatically propose times with Google Calendar -->
+  <head>
+    <meta charset="UTF-8">
+    <title>Create Meeting</title>
+    <script src="js/error-messages.js"></script>
+    <script src="js/time-proposal-util.js"></script>
+    <script src="js/dao-utils.js"></script>
```

Comment:
nit: Alphabetical order

Reply:
I've doing a little bit of [research](https://eager.io/blog/everything-I-know-about-the-script-tag/) into the order that <script> tags need to be in, and it looks like script tags will execute in the order they are loaded, so scripts that are dependent on other scripts should be loaded later (rather than in alphabetical order like in Java).

Because most of our JavaScript are in functions that aren't called until a DOM event happens (by the time when all scripts are loaded) the order in which they appear is less important (so I _can_ rearrange in alphabetical order and no errors occur), but I think because all the <scripts> below error-messages.js use constants defined there, error-messages.js should be loaded first in case of errors that get introduced later when (_assuming that 😅_) changes are made to the code.

Similarly, both dao-utils and time-proposal-utils are used in the <script> tags listed below.

Reply:
Ah okay. Looks like you've thought about it quite a bit so sounds good :)